haskellPackages: update stackage and hackage

sternenseemann opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. You may be able to find an up-to-date Hydra build report at [cdepillabout/nix-haskell-updates-status](https://github.com/cdepillabout/nix-haskell-updates-status).

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/H6migHmKX7xHoTFa) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #176219. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

I'll sign out for this week and will be back mid next week.

sternenseemann committed:
haskellPackages: stackage LTS 19.9 -> LTS 19.10

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

sternenseemann committed:
all-cabal-hashes: 2022-06-04T09:01:11Z -> 2022-06-07T15:13:17Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

sternenseemann committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

github-actions[bot] committed:
Merge master into haskell-updates

cdepillabout committed:
haskellPackages.highlight: unbreak

cdepillabout committed:
haskellPackages.ghc-lib-parser-ex_9_2_1_0: update overrides for this version

Commit:
haskell.packages.ghcjs.ghcjs-base: fix build

The latest aeson doesn't build in ghcjs because it includes text-short
in its dependency tree, which hangs on checkPhase because it uses C FFI.
But since ghcjs-base has an upper bound on aeson<1.6 anyway, just
override it.

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

trofi committed:
haskellPackages.lvmrun: mark as proken

The package fails to build with upstream gcc-10 as:

    ld: heap/bytes.o:(.data.rel.local+0x0): multiple definition of `bytes_ops';
      core/loader.o:(.bss+0x0): first defined here

Upstream report: https://github.com/Helium4Haskell/lvm/pull/4

nomeata committed:
haskellPackages.jsaddle-warp: no longer broken

it builds, so don’t mark it as broken

github-actions[bot] committed:
Merge master into haskell-updates

maralorn committed:
Merge pull request #177313 from nomeata/joachim/jsaddle-warp-unbroken

haskellPackages.jsaddle-warp: no longer broken

maralorn committed:
Merge pull request #176953 from libjared/ghcjs-aeson

haskell.packages.ghcjs.ghcjs-base: fix build

github-actions[bot] committed:
Merge master into haskell-updates

nomeata committed:
haskell.packages.ghcjs.ghcjs-base: 0.2.0.3 → 0.2.1.0

the new upstream release makes it build with aeson-2.0.

github-actions[bot] committed:
Merge master into haskell-updates

cdepillabout committed:
Merge pull request #177276 from trofi/workaround-fno-common-for-haskellPackages.lvmrun

haskellPackages.lvmrun: mark as broken

cdepillabout commented:
Looks like a problem with `hnix` is causing a couple dependencies to fail as well:

### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1766947](https://hydra.nixos.org/eval/1766947) of nixpkgs commit [e6ff544](https://github.com/NixOS/nixpkgs/commits/e6ff544a51e102ebe0849b9e24ee5937c912df0b) as of 2022-06-14 01:14 UTC*

:yellow_circle: **Potential issues**
  * `maintained` jobset failed.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1766947?filter=.aarch64-linux) | 24 | 31 | 5 | 6250 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1766947?filter=.x86_64-darwin) | 64 | 25 | 4 | 6161 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1766947?filter=.x86_64-linux) | 3 | 8 | 4 | 6327 | 
#### Maintained packages with build failure
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179745480) [[:apple::x:]](https://hydra.nixos.org/build/179746884) [[:penguin::x:]](https://hydra.nixos.org/build/179747154) [haskellPackages.hnix](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hnix) @Anton-Latukha
#### Maintained packages with failed dependency
- [ ] [dhall-nix](https://hydra.nixos.org/eval/1766947?filter=dhall-nix) @Gabriel439
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743438) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743348) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746202) [toplevel](https://hydra.nixos.org/eval/1766947?filter=dhall-nix)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744932) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744690) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747477) [haskellPackages](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.dhall-nix)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745380) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746704) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746367) [haskellPackages.dhall-nixpkgs](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.dhall-nixpkgs) @Gabriel439
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315975) [maintained](https://hydra.nixos.org/eval/1766947?filter=maintained) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [nix-linter](https://hydra.nixos.org/eval/1766947?filter=nix-linter) @marsam
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745867) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743854) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743630) [toplevel](https://hydra.nixos.org/eval/1766947?filter=nix-linter)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747253) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744235) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743451) [haskellPackages](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.nix-linter)
- [ ] [update-nix-fetchgit](https://hydra.nixos.org/eval/1766947?filter=update-nix-fetchgit) @expipiplus1 @sorki
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315957) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315977) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315922) [toplevel](https://hydra.nixos.org/eval/1766947?filter=update-nix-fetchgit)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315965) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315935) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/180315959) [haskellPackages](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.update-nix-fetchgit)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744155) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743545) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746306) [haskellPackages.yarn2nix](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.yarn2nix) @sternenseemann
#### Unmaintained packages with build failure
<details><summary>81 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179072015) [[:apple::x:]](https://hydra.nixos.org/build/179063516) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079506) [haskellPackages.di-core](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di-core)  :arrow_heading_up: 8 | 11
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179071185) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179068059) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063826) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 5 | 36
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179747097) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746782) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744135) [haskellPackages.hw-json-simd](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-json-simd)  :arrow_heading_up: 2 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179745918) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746509) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743244) [haskellPackages.hw-simd](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-simd)  :arrow_heading_up: 2 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179289726) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179290108) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179290785) [haskellPackages.flatparse](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.flatparse)  :arrow_heading_up: 2 | 5
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179746327) [[:apple::x:]](https://hydra.nixos.org/build/179746283) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743419) [haskellPackages.fakedata](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.fakedata)  :arrow_heading_up: 2 | 3
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179746353) [[:apple::x:]](https://hydra.nixos.org/build/179745534) [[:penguin::x:]](https://hydra.nixos.org/build/179746815) [haskellPackages.managed-functions](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.managed-functions)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179743898) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179745897) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744251) [haskellPackages.quic](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.quic)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179064849) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179066591) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179076598) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.freetype2)  :arrow_heading_up: 1 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179066416) [[:apple::x:]](https://hydra.nixos.org/build/179077317) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179073445) [haskellPackages.invertible](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.invertible)  :arrow_heading_up: 1 | 5
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179066907) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179063014) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065251) [haskellPackages.long-double](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.long-double)  :arrow_heading_up: 1 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179062753) [[:apple::x:]](https://hydra.nixos.org/build/179069557) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068287) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179066266) [[:apple::x:]](https://hydra.nixos.org/build/179073489) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179077924) [haskellPackages.grab](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.grab)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179078946) [[:apple::x:]](https://hydra.nixos.org/build/179074000) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179071674) [haskellPackages.keep-alive](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.keep-alive)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179062052) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179078236) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179073709) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179070564) [[:apple::x:]](https://hydra.nixos.org/build/179073841) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063192) [haskellPackages.sequence-formats](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.sequence-formats)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179743728) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179747585) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745045) [haskellPackages.swisstable](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.swisstable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179071144) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179078269) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179080527) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179745492) [[:apple::x:]](https://hydra.nixos.org/build/179745291) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744704) [haskellPackages.zip](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.zip)  :arrow_heading_up: 0 | 5
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179078671) [[:apple::x:]](https://hydra.nixos.org/build/179078612) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179069097) [haskellPackages.PyF](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.PyF)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179072055) [[:apple::x:]](https://hydra.nixos.org/build/179078039) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068269) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179747173) [[:apple::x:]](https://hydra.nixos.org/build/179744882) [[:penguin::x:]](https://hydra.nixos.org/build/179744496) [haskellPackages.core-telemetry](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.core-telemetry)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179064212) [[:apple::x:]](https://hydra.nixos.org/build/179067470) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179067662) [haskellPackages.posix-socket](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.posix-socket)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994602) [[:apple::x:]](https://hydra.nixos.org/build/179745342) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994202) [haskellPackages.gi-gdkx11](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gi-gdkx11)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179075182) [[:apple::x:]](https://hydra.nixos.org/build/179068241) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079129) [haskellPackages.hamid](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hamid)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179069659) [[:apple::x:]](https://hydra.nixos.org/build/179071126) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079563) [haskellPackages.hmatrix-morpheus](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hmatrix-morpheus)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179073746) [[:apple::x:]](https://hydra.nixos.org/build/179077643) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179080493) [haskellPackages.huckleberry](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.huckleberry)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994619) [[:apple::x:]](https://hydra.nixos.org/build/179079172) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994199) [haskellPackages.openal-ffi](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.openal-ffi)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179065449) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179072649) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179067032) [haskellPackages.picosat](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179077629) [[:apple::x:]](https://hydra.nixos.org/build/179072654) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065194) [haskellPackages.select](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.select)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179067209) [[:apple::x:]](https://hydra.nixos.org/build/179063690) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068800) [haskellPackages.sysinfo](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.sysinfo)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179071092) [[:apple::x:]](https://hydra.nixos.org/build/179080756) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065681) [haskellPackages.FractalArt](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.FractalArt) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179062993) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179067400) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179073212) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/179064497) [[:apple::x:]](https://hydra.nixos.org/build/179062384) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/179066691) [haskellPackages.bindings-common](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.bindings-common) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179073684) [[:apple::x:]](https://hydra.nixos.org/build/179077115) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179064418) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179290863) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179291122) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179291222) [haskellPackages.comfort-fftw](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.comfort-fftw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179067240) [[:apple::x:]](https://hydra.nixos.org/build/179066776) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063089) [haskellPackages.diskhash](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.diskhash) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179072726) [[:apple::x:]](https://hydra.nixos.org/build/179069153) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063920) [haskellPackages.env-extra](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.env-extra) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179080379) [[:apple::x:]](https://hydra.nixos.org/build/179069971) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179066173) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179080024) [[:apple::x:]](https://hydra.nixos.org/build/179075268) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179073623) [haskellPackages.fudgets](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.fudgets) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743762) [[:apple::x:]](https://hydra.nixos.org/build/179744285) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744321) [haskellPackages.gerrit](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gerrit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179290291) [[:apple::x:]](https://hydra.nixos.org/build/179289980) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179289981) [haskellPackages.ghc-gc-hook](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.ghc-gc-hook) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/179743319) [haskellPackages.gi-gtkosxapplication](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gi-gtkosxapplication) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179994262) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994321) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gnome-keyring) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/179069635) [haskellPackages.gtk-mac-integration](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gtk-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994402) [[:apple::x:]](https://hydra.nixos.org/build/179067045) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994448) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/179080546) [haskellPackages.gtk3-mac-integration](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.gtk3-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994258) [[:apple::x:]](https://hydra.nixos.org/build/179994252) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994596) [haskellPackages.hid](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179072433) [[:apple::x:]](https://hydra.nixos.org/build/179080030) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068258) [haskellPackages.higher-leveldb](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.higher-leveldb) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179773405) [[:apple::x:]](https://hydra.nixos.org/build/179773407) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179773406) [haskellPackages.highlight](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.highlight) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743535) [[:apple::x:]](https://hydra.nixos.org/build/179747090) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745070) [haskellPackages.hinotify-conduit](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hinotify-conduit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179061598) [[:apple::x:]](https://hydra.nixos.org/build/179063833) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179078483) [haskellPackages.hssh](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hssh) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179078447) [[:apple::x:]](https://hydra.nixos.org/build/179064179) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065544) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179074844) [[:apple::x:]](https://hydra.nixos.org/build/179063095) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179078906) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179068045) [[:apple::x:]](https://hydra.nixos.org/build/179079348) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179070084) [haskellPackages.ipcvar](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.ipcvar) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179071989) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179071089) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063380) [haskellPackages.jammittools](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.jammittools) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/179073560) [haskellPackages.kqueue](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.kqueue) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179074199) [[:apple::x:]](https://hydra.nixos.org/build/179064612) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063995) [haskellPackages.leveldb-haskell-fork](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.leveldb-haskell-fork) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179073139) [[:apple::x:]](https://hydra.nixos.org/build/179061940) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179080260) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743996) [[:apple::x:]](https://hydra.nixos.org/build/179743754) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746155) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179069027) [[:apple::x:]](https://hydra.nixos.org/build/179071937) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079130) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179068290) [[:apple::x:]](https://hydra.nixos.org/build/179080076) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179074457) [haskellPackages.nano-cryptr](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.nano-cryptr) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994460) [[:apple::x:]](https://hydra.nixos.org/build/179745227) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994430) [haskellPackages.persistent-pagination](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.persistent-pagination) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179071816) [[:apple::x:]](https://hydra.nixos.org/build/179073643) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179072304) [haskellPackages.phatsort](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.phatsort) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179073647) [[:apple::x:]](https://hydra.nixos.org/build/179080408) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179070022) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179071908) [[:apple::x:]](https://hydra.nixos.org/build/179075532) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079347) [haskellPackages.posix-timer](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.posix-timer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179075838) [[:apple::x:]](https://hydra.nixos.org/build/179065867) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179077987) [haskellPackages.pthread](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.pthread) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179076975) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179067319) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179062948) [haskellPackages.risc386](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.risc386) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179069415) [[:apple::x:]](https://hydra.nixos.org/build/179073373) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179074823) [haskellPackages.scenegraph](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.scenegraph) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994564) [[:apple::x:]](https://hydra.nixos.org/build/179067157) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994543) [haskellPackages.sfml-audio](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.sfml-audio) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179079829) [[:apple::x:]](https://hydra.nixos.org/build/179078716) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179072353) [haskellPackages.shared-memory](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.shared-memory) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179068637) [[:apple::x:]](https://hydra.nixos.org/build/179072695) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179069692) [haskellPackages.skews](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.skews) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179079291) [[:apple::x:]](https://hydra.nixos.org/build/179078891) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068825) [haskellPackages.slugify](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.slugify) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179080373) [[:apple::x:]](https://hydra.nixos.org/build/179072670) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179071968) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179070427) [[:apple::x:]](https://hydra.nixos.org/build/179063366) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179074860) [haskellPackages.tini](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.tini) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179065640) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179067242) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179080300) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/179066662) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179077309) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179066209) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179067680) [[:apple::x:]](https://hydra.nixos.org/build/179064378) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065946) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179075430) [[:apple::x:]](https://hydra.nixos.org/build/179067953) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065799) [haskellPackages.yoga](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.yoga) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179079532) [[:apple::x:]](https://hydra.nixos.org/build/179065669) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179075069) [haskellPackages.zot](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.zot) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179066474) [[:apple::x:]](https://hydra.nixos.org/build/179064085) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179066132) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>47 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179070983) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179072217) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179075865) [haskellPackages.di-handle](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di-handle)  :arrow_heading_up: 6 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179071514) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179068809) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079272) [haskellPackages.di-monad](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di-monad)  :arrow_heading_up: 6 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179061716) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179072623) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179068300) [haskellPackages.di-df1](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di-df1)  :arrow_heading_up: 5 | 8
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747485) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746293) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746091) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 4 | 35
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746491) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746239) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743981) [haskellPackages.BiobaseTypes](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.BiobaseTypes)  :arrow_heading_up: 3 | 21
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743689) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179744861) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745167) [haskellPackages.BiobaseENA](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.BiobaseENA)  :arrow_heading_up: 1 | 18
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179076522) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179063764) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179069376) [haskellPackages.di-polysemy](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di-polysemy)  :arrow_heading_up: 1 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179289701) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179290825) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179289863) [haskellPackages.exon](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.exon)  :arrow_heading_up: 1 | 3
- [ ] [hoogle](https://hydra.nixos.org/eval/1766947?filter=hoogle)  :arrow_heading_up: 1 | 3
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179747540) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179744194) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746071) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1766947?filter=haskell.packages.ghc8107.hoogle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743381) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179745964) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744098) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1766947?filter=haskell.packages.ghc884.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179747595) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179745051) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179747163) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1766947?filter=haskell.packages.ghc902.hoogle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747195) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179743693) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745899) [haskell.packages.ghc923](https://hydra.nixos.org/eval/1766947?filter=haskell.packages.ghc923.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179747023) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179747450) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744848) [haskellPackages](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hoogle)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743449) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179744501) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745958) [haskellPackages.http3](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.http3)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747381) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743188) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745520) [haskellPackages.managed-functions-json](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.managed-functions-json)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743807) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743948) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743591) [haskellPackages.moto](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.moto)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179747395) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746658) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745715) [haskellPackages.wss-client](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.wss-client)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743353) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179747517) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743704) [haskellPackages.BiobaseXNA](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.BiobaseXNA)  :arrow_heading_up: 0 | 17
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747577) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179744159) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746935) [haskellPackages.hw-json-standard-cursor](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-json-standard-cursor)  :arrow_heading_up: 0 | 6
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743663) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179745141) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745597) [haskellPackages.hw-json-simple-cursor](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-json-simple-cursor)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747483) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179743444) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744777) [haskellPackages.BiobaseFasta](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.BiobaseFasta)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179746392) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179745302) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744580) [haskellPackages.hw-dsv](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-dsv)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179074702) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179061581) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179075723) [haskellPackages.di](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.di)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744804) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746711) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746512) [haskellPackages.polysemy-http](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.polysemy-http)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179745539) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745290) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179744723) [haskellPackages.hedgehog-fakedata](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hedgehog-fakedata)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179064709) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179062645) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179079262) [haskellPackages.invertible-hxt](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.invertible-hxt)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179994556) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179290635) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994385) [haskellPackages.align-audio](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.align-audio) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179074482) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179073189) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065292) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743440) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743314) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179747044) [haskellPackages.fakedata-quickcheck](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.fakedata-quickcheck) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179080648) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179062422) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179064448) [haskellPackages.grab-form](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.grab-form) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179067618) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179076454) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179065362) [haskellPackages.harfbuzz-pure](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.harfbuzz-pure) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179070771) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179074263) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179080405) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744317) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746403) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745083) [haskellPackages.hs-swisstable-hashtables-class](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hs-swisstable-hashtables-class) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747125) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746710) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746697) [haskellPackages.hw-simd-cli](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.hw-simd-cli) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179079161) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179061880) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063408) [haskellPackages.kmn-programming](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.kmn-programming) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743649) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745254) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747083) [haskellPackages.managed-functions-http-connector](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.managed-functions-http-connector) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994359) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743537) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994274) [haskellPackages.moto-postgresql](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.moto-postgresql) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179743213) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179747461) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179746172) [haskellPackages.network-messagepack-rpc-websocket](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.network-messagepack-rpc-websocket) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179745948) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179744480) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179743751) [haskellPackages.polysemy-log-di](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.polysemy-log-di) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179994416) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179743407) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994389) [haskellPackages.postgresql-replicant](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.postgresql-replicant) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179073000) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179066926) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179064722) [haskellPackages.rounded-hw](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.rounded-hw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179066839) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179070507) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179074109) [haskellPackages.sequenceTools](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.sequenceTools) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179994579) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179289897) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179994395) [haskellPackages.sound-collage](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.sound-collage) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179070293) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179072703) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179063216) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179745498) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/179746177) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179745170) [haskellPackages.warp-quic](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.warp-quic) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/179079321) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/179080550) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/179064359) [haskellPackages.xbattbar](https://hydra.nixos.org/eval/1766947?filter=haskellPackages.xbattbar) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[amazonka-core](https://packdeps.haskellers.com/reverse/amazonka-core) :arrow_heading_up: 185  
[gogol-core](https://packdeps.haskellers.com/reverse/gogol-core) :arrow_heading_up: 184  
[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[util](https://packdeps.haskellers.com/reverse/util) :arrow_heading_up: 49  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[amazonka](https://packdeps.haskellers.com/reverse/amazonka) :arrow_heading_up: 43  
[accelerate](https://packdeps.haskellers.com/reverse/accelerate) :arrow_heading_up: 42  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 42  
[syb-with-class](https://packdeps.haskellers.com/reverse/syb-with-class) :arrow_heading_up: 42  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[autodocodec](https://packdeps.haskellers.com/reverse/autodocodec) :arrow_heading_up: 34  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 33  
[rank1dynamic](https://packdeps.haskellers.com/reverse/rank1dynamic) :arrow_heading_up: 33  
[distributed-static](https://packdeps.haskellers.com/reverse/distributed-static) :arrow_heading_up: 31  
[language-ecmascript](https://packdeps.haskellers.com/reverse/language-ecmascript) :arrow_heading_up: 31  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[ip](https://packdeps.haskellers.com/reverse/ip) :arrow_heading_up: 29  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[validity-aeson](https://packdeps.haskellers.com/reverse/validity-aeson) :arrow_heading_up: 29  
[text-format](https://packdeps.haskellers.com/reverse/text-format) :arrow_heading_up: 28  
[autodocodec-schema](https://packdeps.haskellers.com/reverse/autodocodec-schema) :arrow_heading_up: 27  
[mmsyn3](https://packdeps.haskellers.com/reverse/mmsyn3) :arrow_heading_up: 27  
[autodocodec-yaml](https://packdeps.haskellers.com/reverse/autodocodec-yaml) :arrow_heading_up: 26  
[crypto-numbers](https://packdeps.haskellers.com/reverse/crypto-numbers) :arrow_heading_up: 25  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[web-routes-th](https://packdeps.haskellers.com/reverse/web-routes-th) :arrow_heading_up: 24  
[ixset-typed](https://packdeps.haskellers.com/reverse/ixset-typed) :arrow_heading_up: 23  
[sydtest](https://packdeps.haskellers.com/reverse/sydtest) :arrow_heading_up: 23  
[crypto-pubkey](https://packdeps.haskellers.com/reverse/crypto-pubkey) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[alg](https://packdeps.haskellers.com/reverse/alg) :arrow_heading_up: 21  
[amazonka-s3](https://packdeps.haskellers.com/reverse/amazonka-s3) :arrow_heading_up: 21  
[mmsyn2](https://packdeps.haskellers.com/reverse/mmsyn2) :arrow_heading_up: 21  
[userid](https://packdeps.haskellers.com/reverse/userid) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[subG](https://packdeps.haskellers.com/reverse/subG) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[fay](https://packdeps.haskellers.com/reverse/fay) :arrow_heading_up: 19  
[harp](https://packdeps.haskellers.com/reverse/harp) :arrow_heading_up: 19  
[hsx2hs](https://packdeps.haskellers.com/reverse/hsx2hs) :arrow_heading_up: 19  
[ixset](https://packdeps.haskellers.com/reverse/ixset) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

sternenseemann committed:
haskellPackages.large-hashable: 2021-11-01 -> 2022-06-10

aeson 2.0 was finally addressed upstream, allowing us to drop a patch.

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

cdepillabout committed:
Merge pull request #177489 from nomeata/joachim/ghcjs-base

haskell.packages.ghcjs.ghcjs-base: 0.2.0.3 → 0.2.1.0

Commit:
haskellPackages: unbreak autodocoded, validity-aeson

cdepillabout committed:
Merge pull request #177887 from m1-s/unbreak_packages

haskellPackages: unbreak autodocodec, validity-aeson

github-actions[bot] committed:
Merge master into haskell-updates

sorki committed:
haskellPackages: update packages maintained by sorki

sorki committed:
haskellPackages: use hnix-store 0.5 for hnix

sorki commented:
Opened #177980 but it needs work.

cdepillabout committed:
haskellPackages.hnix: small refactoring of overrides

cdepillabout committed:
Merge pull request #177980 from sorki/haskell-updates

haskellPackages: use hnix-store 0.5 for hnix, update maintained packages

cdepillabout committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

cdepillabout committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh

cdepillabout committed:
haskellPackages.jsaddle-hello: mark broken on darwin

cdepillabout committed:
tamarin-prover: remove darwin from hydraPlatforms

cdepillabout committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh

cdepillabout committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh
